<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDlg</class>
 <widget class="QDialog" name="MessageDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>客户端消息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>15</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册请求</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>15</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>路况信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>195</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>配送情况</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="RegTable">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>22</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电话</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>邮箱</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="roadTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>22</number>
   </attribute>
   <column>
    <property name="text">
     <string>用户</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>详情</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>370</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QListWidget" name="delvList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectBtn">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>370</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
